charts/evm-rollup/files/genesis/geth-genesis.json

```diff
@@ -26,6 +26,9 @@
         "terminalTotalDifficulty": 0,
         "terminalTotalDifficultyPassed": true,
         "ethash": {},
+        {{- if .Values.config.rollup.genesis.forma }}
```

Comment:
Should we make this more generic and just add a "additional genesis" field that's converted to json?

Reply:
that sounds better.